packages/credo/src/types.ts

```diff
@@ -49,10 +51,19 @@ export const RegisterCredentialDefinitionParams ={
     })
 };
 
+export const ListCredentialDefinitionParams = {}
 
 // connection
 export const CreateInvitationParams = {}
 
 export const ReceiveInvitationParams = {
     invitationUrl: z.string().describe("Provide the invitation url to establish a secure didcomm connection")
-}
\ No newline at end of file
+}
+
+// credential
+export const ConnectionlessCredentialOfferParams = {
+    attributes: z.object({}).describe("Provide the list of attributes published in the schema linked to the provided credentialDefinitionId"),
+    credentialDefinitionId: z.string().startsWith("did:cheqd:").includes("/resources/").describe("DID Url of credentialDefinitionId e.g. did:cheqd:testnet:4769f00d-0af4-472b-aab7-019abbbb8009/resources/5acb3d53-ba06-441a-b48b-07d8c2f129f8. You have the option to list the credential definitionIds with the other tool")
```

Comment:
Guess out of the scope of this PR but should this be hardcoded?

Reply:
Yes, for the agent to have context about the arg